tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -1736,20 +1736,42 @@ def test_alert_and_widget_colliding(default_project: Project) -> None:
 
 
 @django_db_all
-def test_not_event_type_error(default_project: Project) -> None:
+@pytest.mark.parametrize(
+    "query, config_assertion, expected_hashes, expected_condition",
+    [
+        ("event.type:error", False, [], None),
+        ("event.type:default", False, [], None),
+        ('event.type:"error"', False, [], None),
+        ("event.type:transaction", True, ["5367d030", "f7a47137"], None),
```

Comment:
There's no need for a condition since `event.type:transaction` is ignored.

Reply:
should we also check `!event.type=transaction` in the test just to make sure we don't have the same problem pop up again in the other direction?

Reply:
Sounds good. I have added this locally.